tests/PHPStan/Rules/Functions/data/function-call-statement-no-side-effects-phpdoc.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace FunctionCallStatementNoSideEffectsPhpDoc;
+
+function regular(string $a): string {return $a;}
+/**
+ * @phpstan-pure
+ */
+function pure1(string $a): string {return $a;}
+/**
+ * @psalm-pure
+ */
+function pure2(string $a): string {return $a;}
```

Comment:
This file should be separated into two files - one with function definitions, one with their calls.